.gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+stages:
+  - prepare
+  - init
+  - deploy
+
+prepare:
+  image: alpine
+  stage: prepare
+  script:
+    - echo -n $service_account > credentials.json
+  artifacts:
+    paths:
+      - credentials.json
+  tags:
+    - test2
```

Comment:
Investigate the variable type file, it might help removing this step

Reply:
It's appear this error
```
ERROR: (gcloud.auth.activate-service-account) unrecognized arguments:
  "service_account",
  "project_id":
  "stashconsulting",
  "private_key_id":
  "2c6725faf2dd841673a79fd61bdf5507774f66c0",
  "private_key":
  "-----BEGIN
```

Reply:
Due to we can run bash in `stashconsulting/terraform-docker:gcloud-latest` image the `prepare` stage was deleted.